arch/arm/soc/st_stm32/stm32f4/soc.h

```diff
@@ -48,6 +48,12 @@
 #include <stm32f4xx_ll_rng.h>
 #endif
 
+/* For IMG_MANAGER */
+#if defined(CONFIG_SOC_FLASH_STM32)
+#define FLASH_DRIVER_NAME	CONFIG_SOC_FLASH_STM32_DEV_NAME
+#define FLASH_ALIGN		FLASH_WRITE_BLOCK_SIZE
```

Comment:
where is FLASH_ALIGN used?

Reply:
In mcuboot.

Reply:
I added a patch which removes FLASH_ALIGN in Zephyr and just uses FLASH_WRITE_BLOCK_SIZE instead. This involved going back and taking it out of soc.h as was done by @nashif and @nvlsianpu in https://github.com/zephyrproject-rtos/zephyr/pull/1562. Hope that's OK.